### PR TITLE
fix unit tests for linux

### PR DESCRIPTION
**Pull request by reeflog**

### CODE REVIEW

File: src/test/groovy/com/ullink/NuGetSpecTest.groovy [hunk 1]
```diff
@@ -164,8 +164,8 @@ class NuGetSpecTest {
         def msbuildTask = new MSBuildTaskBuilder()
                 .withAssemblyName('bar')
                 .withFrameworkVersion('v3.5')
-                .withArtifact('c:\\folder\\bin\\bar.dll')
-                .withProjectFile('c:\\does not exist')
+                .withArtifact('folder\\bin\\bar.dll')
```
Comment (timotei) [inline src/test/groovy/com/ullink/NuGetSpecTest.groovy]: If you use forward slash it should work on windows too.

File: src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy [hunk 2]
```diff
@@ -33,12 +33,12 @@ class MSBuildTaskBuilder {
     }
 
     public MSBuildTaskBuilder withArtifact(String artifactPath) {
-        artifacts.add(new File(artifactPath))
+        artifacts.add(new File(artifactPath.replace('\\', File.separator)))
```
Comment (timotei) [inline src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy]: I don't this this is such a good idea, is it? You might intentionally have "\" in the path, like when escaping space (at least on linux)

File: src/test/groovy/com/ullink/NuGetSpecTest.groovy [hunk 3]
```diff
@@ -183,9 +183,9 @@ class NuGetSpecTest {
                 <title>foo</title>
             </metadata>
             <files>
-                <file src='c:\\folder\\bin\\bar.dll' target='lib/net35' />
+                <file src='folder\\bin\\bar.dll' target='lib/net35' />
             </files>
-        </package>'''
+        </package>'''.replace('\\', File.separator)
```
Comment (timotei) [inline src/test/groovy/com/ullink/NuGetSpecTest.groovy]: Same, better use forward slash IMO

File: src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy (hunk 2 above)
Comment (reeflog) [inline src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy]: I agree but it's for unit tests. We could find a more global solution when such think will be used.

File: src/test/groovy/com/ullink/NuGetSpecTest.groovy (hunk 3 above)
Comment (reeflog) [inline src/test/groovy/com/ullink/NuGetSpecTest.groovy]: I've tried but the file.path implementation in Windows is with '\' and the implementation in Linux is with '/'.

File: src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy (hunk 2 above)
Comment (reeflog) [inline src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy]: I've removed it. It was not needed. Thanks for the feedback.

File: src/test/groovy/com/ullink/NuGetSpecTest.groovy (hunk 1 above)
Comment (reeflog) [inline src/test/groovy/com/ullink/NuGetSpecTest.groovy]: Yes, working.

File: src/main/groovy/com/ullink/NuGetSpec.groovy [hunk 4]
```diff
@@ -65,7 +65,7 @@ class NuGetSpec extends Exec {
         def final msbuildTaskExists = project.tasks.findByName('msbuild') != null
         def final packageConfigFileName = 'packages.config'
 
-        def GPathResult root = new XmlSlurper(false, false).parseText(nuspecString)
+        GPathResult root = new XmlSlurper(false, false).parseText(nuspecString)
```
Comment (reeflog) [inline src/main/groovy/com/ullink/NuGetSpec.groovy]: I've removed the un-necessary 'def' as well (remark from another pull request).

**Pull request merged**